src/java/org/apache/cassandra/security/SSLFactory.java

```diff
@@ -351,9 +194,16 @@ public static void checkCertFilesForHotReloading(EncryptionOptions.ServerEncrypt
         if (!isHotReloadingInitialized)
             throw new IllegalStateException("Hot reloading functionality has not been initialized.");
 
-        logger.debug("Checking whether certificates have been updated {}", hotReloadableFiles);
+        logger.debug("Checking whether certificates have been updated {} and {}",
+                     serverOpts.sslContextFactoryInstance.getClass().getName(), clientOpts.sslContextFactoryInstance.getClass().getName());
+
+        boolean serverSslContextShouldReload = serverOpts == null ? false :
```

Comment:
Both of these might be simplified similarly as for hot loading files case, like serverOpts != null && serverOpts...shouldReload();